change/@itwin-tree-widget-react-ca23f996-6160-4e15-9918-2444a229add8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added ability to track performance of `TreeWidget` component features.",
```

Comment:
I think this should mention the requirement for `@itwin/presentation-components` to be at `5.1` for the reporting to work.

Reply:
Mentioned the requirement.